data/mainnet/tokens.js

```diff
@@ -45,6 +45,109 @@ module.exports = {
     ],
     cmcRewardTokenSymbols: ['FARM', 'WETH'],
   },
+  note: {
+    chain: CHAINS_ID.ETH_MAINNET,
+    logoUrl: './icons/note.png',
+    tokenAddress: addresses.NOTE,
+    decimals: '8',
+    vaultAddress: null,
+    priceFunction: {
+      type: GET_PRICE_TYPES.COINGECKO_ID,
+      params: ['notional-finance'],
+    },
+  },
+  notional_DAI: {
+    category: VAULT_CATEGORIES_IDS.NOTIONAL,
```

Comment:
Let's also add the stablecoin category. It should work if you add multiple categories in a list. `[VAULT_CATEGORIES_IDS.NOTIONAL, VAULT_CATEGORIES_IDS.STABLECOINS]`.

Please do this for both nDAI and nUSDC.